### PR TITLE
New version: Notepad++.Notepad++ version 8.3.3

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/54453)

### CODE REVIEW

File: manifests/n/Notepad++/Notepad++/8.3.3/Notepad++.Notepad++.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+UpgradeBehavior: install
+ReleaseDate: 2022-03-14
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/notepad-plus-plus/notepad-plus-plus/releases/download/v8.3.3/npp.8.3.3.Installer.arm64.exe
+  InstallerSha256: 00F44177F99D368A608C814DF24EB381308A8EE6F4E94D7982F2282277252C5A
+- Architecture: x86
+  InstallerUrl: https://github.com/notepad-plus-plus/notepad-plus-plus/releases/download/v8.3.3/npp.8.3.3.Installer.exe
+  InstallerSha256: 5B7E0B9218CD07D380D894B773A4A835FAD01EA7B5C7BD49B6D48B31415626A7
+- Architecture: arm64
```
Comment (ItzLevvie) [inline manifests/n/Notepad++/Notepad++/8.3.3/Notepad++.Notepad++.installer.yaml:22]:
```suggestion
- Architecture: x64
```

File: manifests/n/Notepad++/Notepad++/8.3.3/Notepad++.Notepad++.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2022-03-14
+Installers:
+- Architecture: x64
```
Comment (ItzLevvie) [inline manifests/n/Notepad++/Notepad++/8.3.3/Notepad++.Notepad++.installer.yaml:16]:
```suggestion
- Architecture: arm64
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Review by derbaumi — APPROVED** [review]
Looking good.

**Pull request merged**